src/util/logging/QueryLogger.ts

```diff
@@ -0,0 +1,64 @@
+import { Logger, QueryRunner } from "typeorm";
+import { LoggerOptions } from "typeorm/logger/LoggerOptions";
+import { PlatformTools } from "typeorm/platform/PlatformTools";
+
+export class QueryLogger implements Logger {
+
+  constructor(private options?: LoggerOptions) {}
+
+  protected writeToFile(message: string) {
+    if (!(message.endsWith("\n"))) message += "\n";
+    const basePath: string = PlatformTools.load("app-root-path").path;
+    PlatformTools.appendFileSync(basePath + "/hub.log", message);
```

Comment:
Could you please extract the filename to an environment variable?

Reply:
Added an environment variable called "HUB_LOG_FILE_NAME"